proof/taproot.go

```diff
@@ -579,3 +630,64 @@ func (p TaprootProof) DeriveByTapscriptProof() (*btcec.PublicKey, error) {
 	}
 	return p.TapscriptProof.DeriveTaprootKeys(p.InternalKey)
 }
+
+// AddExclusionProofs adds exclusion proofs to the base proof for each P2TR
+// output in the given PSBT that isn't an anchor output itself. To determine
+// which output is the anchor output, the passed isAnchor function should
```

Comment:
nit: We can have multiple anchor outputs, and `isAnchor` may return true for multiple indexes, such as in `createProofs()`. Could be:

`return true for any output index that houses a Taro commitment root.`